javascript/javascript3/week1/GiphyApi/main.js

```diff
@@ -0,0 +1,42 @@
+//Weather app
+//Fetch weather data from a city
+
+const searchForm = document.getElementById('search-form');
+const searchInput = document.getElementById('input');
+const inpunNum = document.getElementById('number');
+const resultsEl = document.getElementById('gif-img');
+
+searchForm.addEventListener('submit', function(e) {
+   e.preventDefault();
+   const keyword = searchInput.value;
+   const num = inpunNum.value;
+   giphySearch(keyword, num);
+});
+
+function giphySearch(keyword, num) {
+   const key = 'sm8PT5ioGhyR8AEZDFMCLMqGERfwzMSO';
+   let api2 = `https://api.giphy.com/v1/gifs/search?api_key=${key}&q=${keyword}&limit=${num}&offset=0&rating=G&lang=en`;
+   fetch(api2)
+      .then(function(res) {
+         return res.json();
+      })
+      .then(function(json) {
```

Comment:
please define a new function and use it as a callback! never do it all within the `then` scope